release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1446,6 +1441,11 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
 ==== External provisioner for AWS EFS (Technology Preview) feature has been removed
 The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview feature has been removed and is no longer supported.
 
+[id="ocp-4-6-tls-common-name"]
+==== TLS verification falling back to the Common Name field
```

Comment:
suggest to add keyword 'removed', like 'TLS verification falling back to the Common Name field removed'